Add `Not available under this OS` error documentation to `faq.md` under General (#285)
kmonad/kmonad: doc/faq.md

in the hardware, before any event is ever registered with the operating system,
therefore KMonad has no way to 'get' at any of those events. This means that we
cannot remap them in any way.

### Q: When I run KMonad I get error `Not available under this OS`

> A: This error occurs when there is OS-specific options in the used configuration

Reviewer: This error occurs when there are OS-specific options in the used configuration

> A: This error occurs when there is OS-specific options in the used configuration

Author: Fixed in https://github.com/kmonad/kmonad/pull/285/commits/902a702def7967a4b4116d3ce401f6877043a739

> file.  Usually this happens when you are on windows and try running the tutorial

Reviewer 2: ```suggestion
file. Usually this happens when you are on windows, try to run the tutorial
```

> file and do not comment, or even delete, the Linux options in `defcfg` and

Reviewer 2: perhaps even "comment out", to be extra clear

> file and do not comment, or even delete, the Linux options in `defcfg` and

Author: Resolved in https://github.com/kmonad/kmonad/pull/285/commits/6599b1c946ee8a2e3c7d2162e2c538b572227bcf

uncomment the Windows options. Nevertheless, this still can happen on other
operating systems, the error message changes slightly based on the operating
system (e.g. `Not available under this OS: LowLevelHookSource`, `Not available

> under this OS: DeviceSource`), but they all starts with `Not available under

Author: Resolved in https://github.com/kmonad/kmonad/pull/285/commits/29a13654f862b46f0ac409c5e7411f4e9a29211a
